src/classes/grade.js

```diff
@@ -19,9 +22,35 @@ class Grade {
         this.domain = utils.getDomain(domain) // strip the subdomain. Fixes matching tosdr for eg encrypted.google.com
         this.isaMajorTrackingNetwork = this.isaMajorTrackingNetwork()
         this.tosdr = this.getTosdr()
+        this.polisis = this.getPolisis()
         this.trackersByUrl = {}
     }
 
+    getPolisis () {
+        let result
+
+        polisisRegexList.some(polisisRE => {
+            let match = polisisRE.exec(this.domain)
+            if (match) {
+                let polisisData = polisis[match[0]]
+
+                if (!polisisData) return
+
+                let numGood = Object.keys(polisisData.good).length
+                let numBad = Object.keys(polisisData.bad).length
+
+                if (numGood && !numBad) {
+                    result = -1
+                } else if (numBad) {
+                    result = 1
+                }
```

Comment:
The algorithm I went for is:

- `-1` to the grade if there's only good privacy policy remarks
- `+1` to the grade if there's any bad privacy policy remarks